client/Window.h.orig

```diff
@@ -5,10 +5,7 @@
 #include "shader.h"
 #include "Character.h"
 #include "EnvElement.h"
-<<<<<<< HEAD
 //#include "../common/constants.h"
-=======
->>>>>>> main
```

Comment:
This parts seem to be commented in the main thread before merging, so I just left it commented here

Reply:
Same for the following files